index.d.ts

```diff
@@ -14,6 +14,7 @@ export type Props = {
   onEnd?: () => void;
   onProgress?: ({ currentTime: number }) => void;
   style?: ViewStyle;
+  resizeAspectFill: boolean;
```

Comment:
This prop is optional. Please add `?`

Reply:
Thank you for pointing this out. I fixed it.

Reply:
Thank you.
Is it possible to implement Android?

> Thank you for contributing.
> For Android, I think that fixing this part can provide the same function.
> Would you please fix it ?
> https://github.com/manse/react-native-brightcove-player/blob/master/android/src/main/java/jp/manse/BrightcovePlayerView.java#L233-L242

Reply:
Sure thing. It may be possible. I will give it a try.